### PR TITLE
Gradle installation note [9827]

### PR DESCRIPTION
**Pull request by imontesino**
Signed-off-by: Ignacio Montesino <ignaciomontesino@eprosima.com>

### CODE REVIEW

File: docs/installation/sources/sources_linux.rst [hunk 1]
```diff
@@ -318,6 +318,15 @@ Once the requirements above are met, compile *Fast DDS-Gen* by following the ste
...
     gradle assemble
 
+.. note::
+
+    If errors occur during compilation or you do not wish to install gradle, executable script is included which will
+    download a temporary gradle for the compilation step.
+    
+    .. code-block:: bash
+
+        ./gradlew assebmle
```
Comment (JLBuenoLopez-eProsima) [inline docs/installation/sources/sources_linux.rst:328]:
```suggestion
        ./gradlew assemble
```

File: docs/installation/sources/sources_linux.rst [hunk 2]
```diff
@@ -318,6 +318,15 @@ Once the requirements above are met, compile *Fast DDS-Gen* by following the ste
     cd Fast-DDS-Gen
     gradle assemble
 
+.. note::
+
+    If errors occur during compilation or you do not wish to install gradle, executable script is included which will
```
Comment (JLBuenoLopez-eProsima) [inline docs/installation/sources/sources_linux.rst:323]:
```suggestion
    If errors occur during compilation or you do not wish to install gradle, an executable script is included which will
```

File: docs/installation/sources/sources_linux.rst [hunk 3]
```diff
@@ -318,6 +318,15 @@ Once the requirements above are met, compile *Fast DDS-Gen* by following the ste
...
+    download a temporary gradle for the compilation step.
```
Comment (JLBuenoLopez-eProsima) [inline docs/installation/sources/sources_linux.rst:324]:
```suggestion
    download gradle temporarily for the compilation step.
```

File: docs/installation/sources/sources_mac.rst [hunk 4]
```diff
@@ -334,6 +334,14 @@ with Homebrew it would be running the command:
 
         brew install gradle
 
+.. note::
+
+    If errors occur during compilation or you do not wish to install gradle, executable script is included which will
```
Comment (JLBuenoLopez-eProsima) [inline docs/installation/sources/sources_mac.rst:339]:
```suggestion
    If errors occur during compilation or you do not wish to install gradle, an executable script is included which will
```

File: docs/installation/sources/sources_mac.rst [hunk 5]
```diff
@@ -334,6 +334,14 @@ with Homebrew it would be running the command:
...
+    download a temporary gradle for the compilation step.
```
Comment (JLBuenoLopez-eProsima) [inline docs/installation/sources/sources_mac.rst:340]:
```suggestion
    download gradle temporarily for the compilation step.
```

File: docs/installation/sources/sources_mac.rst [hunk 6]
```diff
@@ -334,6 +334,14 @@ with Homebrew it would be running the command:
...
+    
+    .. code-block:: bash
+
+        ./gradlew assebmle
```
Comment (JLBuenoLopez-eProsima) [inline docs/installation/sources/sources_mac.rst:344]:
```suggestion
        ./gradlew assemble
```

**Review by JLBuenoLopez-eProsima — CHANGES_REQUESTED** [review]
Have you checked if the script works correctly in MacOS platform?

**Comment by imontesino** [comment]
Waiting on eProsima/Fast-DDS-Gen/pull/46 for correct behavior on macOS.

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/651/)

**Review by JLBuenoLopez-eProsima — DISMISSED** [review]
LGTM with green CI

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/652/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/653/)

**Review by EduPonz — APPROVED** [review]
LGTM

**Pull request merged**